modules/dqt/jsx/react.filterBuilder.js

```diff
@@ -9,7 +9,7 @@
 
 import React, {Component} from 'react';
 import ModalImportCSV from './react.importCSV';
-import {getSessions} from '../js/arrayintersect';
+import {getSessions, enumToArray} from '../js/arrayintersect';
```

Comment:
@ridz1208 this import fixes the "Fixes the unable to load a saved query bug. One of the functions needed wasn't being imported in the react.app.js" bug.